ci/plugins/cloudtest/hooks/command

```diff
@@ -27,6 +27,11 @@ fi
 
 date +"%Y-%m-%d %H:%M:%S" > step_start_timestamp
 
+ci_collapsed_heading "kail: Clean up running instances, start a new instance"
+run pkill -9 -f "^kail$" || true
+run rm -f "kail-output.log"
+run kail > "kail-output.log"
```

Comment:
This needs to run inside the `ci-builder` container, but the `run` bash function does not provide that -- you need to use `bin/ci-builder run stable ...` for the purpose.

Reply:
Thank you!